src/Sylius/Bundle/AdminBundle/Resources/config/grids/payment.yml

```diff
@@ -0,0 +1,67 @@
+sylius_grid:
+    grids:
+        sylius_admin_payment:
+            driver:
+                name: doctrine/orm
+                options:
+                    class: "%sylius.model.payment.class%"
+            fields:
+                id:
+                    type: string
+                    label: sylius.ui.id
+                number:
+                    type: twig
+                    label: sylius.ui.number
+                    path: order
+                    options:
+                        template: "@SyliusAdmin/Order/Grid/Field/number.html.twig"
+                channel:
+                    type: twig
+                    label: sylius.ui.channel
+                    path: order.channel
+                    options:
+                        template: "@SyliusAdmin/Order/Grid/Field/channel.html.twig"
+                customer:
+                    type: twig
+                    label: sylius.ui.customer
+                    path: order.customer
+                    options:
+                        template: "@SyliusAdmin/Order/Grid/Field/customer.html.twig"
+                state:
+                    type: twig
+                    label: sylius.ui.state
+                    options:
+                        template: "@SyliusUi/Grid/Field/state.html.twig"
+                        vars:
+                            labels: "@SyliusAdmin/Order/Label/PaymentState"
+            filters:
+                state:
+                    type: select
+                    label: sylius.ui.state
+                    form_options:
+                        choices:
+                            sylius.ui.new: New
```

Comment:
Do we want to filter only over these two states? Maybe we should add a separate task for rest of states? /cc @CoderMaggie